### ISSUE TITLE
[TASK] Meet and try to onboard Rami Kadi to NEAR

### ISSUE THREAD

**Issue by BenKurrek** [near/devrel#383]
## What is this for? 
- Rami Kadi is an extremely well-known fashion designer based out of Lebanon. He wants to integrate NFTs into his projects and I managed to shill NEAR enough for him to want to use our chain. He would like to build out on either Paras of Fayyr for his upcoming collection in Paris (January 2022).  

## Who needs to be involved?
- @BenKurrek 

## Tasks to complete (can return to [x] these)
- [x] meeting with Joud (his advisor). 
- [x] send a detailed description of all the possible NFT platforms
- [ ] work on onboarding Rami to the platform of his choice. 

## What is the t-shirt size of this? (range: x-small is 1 hour, x-large is over a month of work)
- [ ] x-small
- [ ] small
- [X] medium
- [ ] large
- [ ] x-large

## What priority is this? (remove what doesn't apply)
- [ ] P0 (EMERGENCY: drop everything to put out the fire)
- [ ] P1 (Critical: docs, bugs, or features that block work or dev/app success)
- [X] P2 (Important: known docs, feature requests, or known bugs)
- [ ] P3 (Nice-to-have: docs or features that improve a flow)

**Comment by BenKurrek** [comment]
Sent a follow up on Friday following our meeting the week before. Waiting on his response.
